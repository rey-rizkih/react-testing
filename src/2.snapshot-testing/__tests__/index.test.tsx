import { render } from "@testing-library/react";

import Component from "..";

describe("Snapshot Testing", () => {
  test("Should match snapshot", () => {
    // 1. Render Component
    const { container } = render(<Component />);

    // 2. Make a snapshot or compare
    expect(container).toMatchSnapshot();
  });
});
