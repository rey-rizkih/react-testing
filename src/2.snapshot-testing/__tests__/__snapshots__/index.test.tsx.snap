// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot Testing Should match snapshot 1`] = `
<div>
  <div
    aria-label="Component"
  >
    component
  </div>
</div>
`;
